rust-runtime/aws-smithy-types/src/byte_stream.rs

```diff
@@ -248,7 +248,7 @@ pin_project! {
     /// 3. **From an `SdkBody` directly**: For more advanced / custom use cases, a ByteStream can be created directly
     /// from an SdkBody. **When created from an SdkBody, care must be taken to ensure retriability.** An SdkBody is retryable
     /// when constructed from in-memory data or when using [`SdkBody::retryable`](crate::body::SdkBody::retryable).
-    ///     ```no_run
```

Comment:
was this required?

Reply:
It might have been something about the host I pulled the repo down on, but it was still compiling for some reason. This was the only way I could get it to pass the initial build (locally)